docs/usage.md

```diff
@@ -150,11 +150,48 @@ public class SimpleMySQLTest {
 }
 ```
 
-### JUnit 5
+### Jupiter / JUnit 5
+
+#### Extension
+
+Jupiter integration is provided by means of the `@Testcontainers` annotation.
+  
+The extension finds all fields that are annotated with `@Container` and calls their container lifecycle 
+methods (only `@Startable` fields are supported). Containers declared as static fields will be shared between test 
```

Comment:
> only `@Startable` fields are supported

`Startable` isn't an annotation... Maybe this could be rephrased:

```suggestion
methods (methods on the `Startable` interface). Containers declared as static fields will be shared between test 
```

Reply:
true, sorry :D